src/Components/Pages/SingIn/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './style.css';
+
+const SignIn = () => {
```

Comment:
There is a weird bug: if you submit the form, the updates made to cards are not longer accessible and when you navigate back to the home page - data are reloaded. Please fix.